Blocstagram/BLCImagesTableViewController.m

```diff
@@ -88,7 +88,16 @@ - (CGFloat) tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexP
 
 }
 
+- (void) tableView:(UITableView *)tableView commitEditingStyle:(UITableViewCellEditingStyle)editingStyle forRowAtIndexPath:(NSIndexPath *)indexPath {
+    
+    if (editingStyle == UITableViewCellEditingStyleDelete) {
+        [self.images removeObjectAtIndex:indexPath.row];
+        [tableView reloadData];
```

Comment:
Ideally, you should not call reloadData but tell the tableView to just delete this one cell. Check the documentation in UITableView for details.